deployment/ccip/changeset/state.go

```diff
@@ -62,6 +62,7 @@ import (
 
 	"github.com/smartcontractkit/chainlink-ccip/chains/evm/gobindings/generated/v1_6_0/fee_quoter"
 
+	"github.com/smartcontractkit/chainlink-ccip/chains/evm/gobindings/generated/latest/don_id_claimer"
```

Comment:
If this contract exists in a versioned folder, we should use that. i.e. `"github.com/smartcontractkit/chainlink-ccip/chains/evm/gobindings/generated/v1_6_1/don_id_claimer"`

If not, we can fix that later.

Reply:
currently set as latest. I'll move it to 1.6.1 later

Reply:
Please create a ticket for this please